wp-content/themes/pub/wporg-learn-2020/css/customiser.css

```diff
@@ -0,0 +1,159 @@
+thead th {
```

Comment:
I think we should delete this file. We have a record of it's contents on the issue

Reply:
I removed the file